api/versions.py

```
@@ -0,0 +1,95 @@
+        """
+        qs = RecordVersion.objects.filter(resource_type=resource_type, resource_id=resource_id).order_by('version')
+        data = [{
+            "version": v.version,
+            "prev_version": v.prev_version,
+            "changed_at": v.changed_at,
+            "diff": v.diff,
+            "meta": v.meta,
+        } for v in qs]
+        return Response(data)
+
+from rest_framework import viewsets, status, serializers
```

Ruff (0.12.2)

Module level import not at top of file

(E402)